runtime/src/in_mem_accounts_index.rs

```diff
@@ -563,6 +559,51 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         }
     }
 
+    /// return true if item was newly added to index
+    fn upsert_on_disk(
+        &self,
+        vacant: VacantEntry<K, AccountMapEntry<T>>,
+        new_entry: PreAllocatedAccountMapEntry<T>,
+        new_entry_zero_lamports: &mut bool,
```

Comment:
It doesn't look like this parameter is read during the fn, just written. Could this be moved to a return value instead?

Edit: Looks like `is_zero_lamports()` could be called from before/outside this function, so maybe this param/return doesn't need to be here at all.